angularSTEP/src/app/app.module.ts

```diff
@@ -1,5 +1,17 @@
 import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
+<<<<<<< HEAD
```

Comment:
The way the "merge" format works is `<<<< HEAD` indicates the beginning of the version of the file from (I might have this backwards) the tip of the branch that was merged